jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -21,11 +22,11 @@
                 - culprits
     scm:
         - git:
-            url: https://github.com/kubernetes/kubernetes
+            url: '{url}'
```

Comment:
What about downstream users that want to use this Jenkins repository to set up tests on their feature fork? Why require them to mangle the job template when we have parameters? This is a pretty common use-case- myself in the last two weeks being one of them.